### PR TITLE
feat: About page

### PR DESCRIPTION
**Pull request by lfaucon**
* Add About page describing the project and the partners and contributors
* New navigation option "About" in sidebar
* Reoganised Routes into categories
* Renamed "about" into "wiki" in Sidebar Footer
* Moved public data download to About page

### CODE REVIEW

File: frontend/src/pages/about/About.tsx [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+  name: string;
+  image: string;
+  institution: string;
+  role: string;
+  description: string;
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.card} style={{ maxWidth: 320 }}>
```
Comment (amatissart) [inline frontend/src/pages/about/About.tsx:46]: `classes.card` is mostly redundant with the built-in style of `<Card>`.
```suggestion
    <Card style={{ maxWidth: 320, padding: '16px' }}>
```

File: frontend/src/pages/about/About.tsx [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+              and studying the Tournesol algorithms.
+            </Typography>
+          </ContentBox>
+        </Grid>
+
+        <Grid item xs={12} sm={4} className={classes.container}>
+          <PeopleCard
+            name="Adrien Matissart"
+            image="/people/Adrien.jpeg"
+            institution="School ID, EPFL"
```
Comment (amatissart) [inline frontend/src/pages/about/About.tsx:206]: Do you mean `IC School`? :wink:

File: frontend/src/pages/about/About.tsx [hunk 3]
```diff
@@ -0,0 +1,310 @@
...
+            institution="MSCI, Inc."
+            role="Treasurer"
+            description="Dr. in Computer Science, Software engineer"
+          />
+        </Grid>
+
+        <Grid item xs={12} sm={4} lg={3} className={classes.container}>
+          <PeopleCard
+            name="Aidan"
+            image="/people/Aidan.jpg"
```
Comment (amatissart) [inline frontend/src/pages/about/About.tsx:176]: Typo in the filename:
```suggestion
            image="/people/aidan.jpg"
```

The image is also very large (> 1MB) and should be reduced/compressed.

File: frontend/src/pages/about/About.tsx [hunk 4]
```diff
@@ -0,0 +1,310 @@
...
+            </Typography>
+            <Typography paragraph>
+              Possible thanks to our users ... Finally, we would like to thank
+              all of the volunteers who compared videos on Tournesol. We count
+              so far about 2500 users who compared 9000 times more than 3000
+              videos.
+            </Typography>
+          </ContentBox>
+        </Grid>
+      </Grid>
```
Comment (amatissart) [inline frontend/src/pages/about/About.tsx:297]: Would it make sense to explain this in a dedicated page, where the content of the dataset could be described in detail?

**Review by amatissart — COMMENTED** [review]
Some images are no longer used: `GitHub_Logo.png` and `Octocat.png`

File: frontend/src/pages/about/About.tsx (hunk 4 above)
Comment (lfaucon) [reply frontend/src/pages/about/About.tsx:297]: Yes, I was quite confused about the content of this "About" page, which seems to be mostly "Who contributes to Tournesol" page. Let's wait a bit until I get a final version, then discuss it when we have a more concrete thing to look at.

File: frontend/src/pages/about/About.tsx [hunk 5]
```diff
@@ -0,0 +1,346 @@
...
+        container
+        className={classes.root}
+        style={{ background: '#1282B2', color: 'white' }}
+      >
+        <Grid item xs={12} className={classes.container}>
+          <ContentBox>
+            <Typography variant="h1">Who builds Tournesol?</Typography>
+            <Typography paragraph>
+              Tournesol seeks to be an extremely transparent project. All
+              contributions to the project are openly visible on our github,
```
Comment (amatissart) [inline frontend/src/pages/about/About.tsx:189]:
```suggestion
              contributions to the project are openly visible on our GitHub,
```

**Review by amatissart — APPROVED** [review]

**Pull request merged**